db/migrate/20180123184510_downcase_username_email.rb

```diff
@@ -0,0 +1,12 @@
+class DowncaseUsernameEmail < ActiveRecord::Migration
+  class TokenAuthenticateMe::User < ActiveRecord::Base
+  end
+
+  def change
+    TokenAuthenticateMe::User.find_each do |user|
```

Comment:
This is a oneway transformation due to losing the original in the downcase. As such this should be in an`up` and the down should `raise ActiveRecord::IrreversibleMigration`